### PR TITLE
docker-machine: add launchd script

### PR DESCRIPTION
**Pull request by acareccia**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/docker-machine.rb [hunk 1]
```diff
@@ -36,6 +36,43 @@ def install
     end
   end
 
+  def caveats
```
Comment (DomT4) [inline Formula/docker-machine.rb]: Unless these caveats are Homebrew-specific we don't want them, really. We try to keep general usage guidelines with upstream.

File: Formula/docker-machine.rb [hunk 2]
```diff
@@ -36,6 +36,43 @@ def install
...
+
+  def plist; <<-EOS.undent
+     <?xml version="1.0" encoding="UTF-8"?>
+     <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+     <plist version="1.0">
+       <dict>
+         <key>EnvironmentVariables</key>
+         <dict>
+             <key>PATH</key>
+             <string>/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin</string>
```
Comment (DomT4) [inline Formula/docker-machine.rb]: You presumably want `#{HOMEBREW_PREFIX}/bin` rather than `/usr/local/bin`. Homebrew isn't always in `/usr/local`.

**Comment by acareccia** [comment]
Done, thank you Dominyk

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/d2b90e146eb33ffb7547dc2300f8044b42e8b802. Thank you for your contribution to Homebrew @acareccia; we appreciate it! :smiley_cat:

**Pull request closed without merging**